spring-content-autoconfigure/src/test/java/org/springframework/content/fs/boot/ContentFilesystemAutoConfigurationTest.java

```diff
@@ -117,7 +116,7 @@ FileSystemResourceLoader fileSystemResourceLoader() {
 
 	}
 
-	@Entity
+	@Entity(name = "TestEntity")
```

Comment:
Why did you add this?  I presume the table name by default would be test_entity, or something similar.  Was there a reason you overrode it to use camel casing?

Reply:
Have no clue, at start I did many things following tutorials.

Reply:
:-) OK.  Let's clean it up then